### ISSUE TITLE
Unify the screen reader bugs document

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/phet-info#246]
We track screen reader bugs in accessibility-bugs.md. We would like to add information about expected behaviors with screen readers.

For example, we recently discovered that VO does not read headings when a screen reader when landing on it. We want a place where we can easily describe behavior, so the team has a place to go to know what is expected.
